src/utils/lambda.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyResult } from "aws-lambda";
```

Comment:
Also eslint rule can be setup to have the same approach for quotes across all application, in some imports you have `"`, in others `'`

Reply:
Done